app/.idea/misc.xml

```diff
@@ -3,6 +3,7 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/Users/oxter/Projects/AndroidStudio/ruffle-android-ox/app/ruffle/src/main/res/layout/activity_main.xml" value="0.12360688956433637" />
```

Comment:
This line looks to be specific to your system, so I would rather not add it to the repo as is.

Reply:
That's my bad, shall I close this pull request and redo it? I can also remove the comment block above.

Reply:
If you're familiar with git, you can also just rearrange the commits in this branch so that only the changes you intend to submit are in it. Then after force-pushing this branch into your fork, the PR will update as well, without having to create a new one.

Reply:
I'm not very well versed, but I think I can figure out how to do that. I'll let you know if I run into any trouble.